internal/git/commands.go

```diff
@@ -852,20 +858,25 @@ func (self *Commands) UnstageAll(runner gitdomain.Runner) error {
 }
 
 func (self *Commands) currentBranchDuringRebase(querier gitdomain.Querier) (gitdomain.LocalBranchName, error) {
-	output, err := querier.QueryTrim("git", "branch", "--list")
+	gitDir, err := self.gitDirectory(querier)
 	if err != nil {
-		return "", err
+		return gitdomain.LocalBranchName(""), err
```

Comment:
In Go, constants without type are untyped. When using an untyped constant expression, it takes on the type of the variable it is assigned to. So `""` will result in a `gitdomain.LocalBranchName("")` here. https://go.dev/blog/constants provides some background.